doc/modules/changes/20190524_bangerth

```diff
@@ -0,0 +1,6 @@
+Remove: The
+SimulatorAccessor::simulator_is_initialized() function has been
```

Comment:
what about source/material_model/drucker_prager.cc ?

Reply:
I had replaced the call to the function in the file you mention.